lib/elixir_segment_api.ex

```diff
@@ -14,8 +14,13 @@ defmodule SegmentAPI do
     -> SegmentAPI.track(bad, bad, bad)
       # {:error, "invalid poison"}
   """
-  def track(event, userId, properties) do
-    case Poison.encode(%{event: event, userId: userId, properties: properties, context: context()}) do
+  def track(event, user_id, properties) do
```

Comment:
Segment requires camelCase for the `userId` key. Not sure which is preferred in case like this: mixing case to use snake_case whenever possible, or just using camelCase everywhere.

Reply:
user_id here is an elixir variable and the naming convention is snake case :D

We can translate to camel for the service at delivery time